### PR TITLE
Add tests in utils

### PR DESCRIPTION
**Pull request by ClSlaid**
add a few tests in chaosd/pkg/utils
also fixed typo in path.go
well maybe this is way too simple

### CODE REVIEW

File: pkg/utils/cidr.go [hunk 1]
```diff
@@ -20,8 +20,14 @@ import (
 
 // IPToCidr converts from an ip to a full mask cidr
 func IPToCidr(ip string) string {
-	// TODO: support IPv6
-	return ip + "/32"
+	// TODO: support IPv6 correctly
```
Comment (WangXiangUSTC) [inline pkg/utils/cidr.go:23]: do you mean the support of IPV6 is not correctly now? 😃

Comment (ClSlaid) [reply pkg/utils/cidr.go:23]: I don't really consider merely appending a "/32" or "/128" to an IP address a correct to convert IP to CIDR. As I know a CIDR more likely refer to a *Range* while IP is to a *Dot*.

Comment (WangXiangUSTC) [reply pkg/utils/cidr.go:23]: ok, we can fix it later

**Review by WangXiangUSTC — APPROVED** [review]
thanks, LGTM

**Pull request merged**